op_crates/web/08_text_encoding.js

```diff
@@ -105,7 +105,7 @@
     if (rem === 1 || /[^+/0-9A-Za-z]/.test(s)) {
       throw new DOMException(
         "The string to be decoded is not correctly encoded",
-        "DataDecodeError",
+        "DOMException",
```

Comment:
The `atob()` should display a `DOMException` error when the parameter is not base64 encoded data.
In the current testcase, since `foobar!!` is not base64 encoded data, it is correct for an exception to occur.

But, In [MDN](https://developer.mozilla.org/en-US/docs/Web/API/DOMException) docs, The second property of `DOMException` is the name of the error. 

So, @gorogoroumaru, As mentioned by @rchougule, It seems more reasonable for us to rename the error to `InvalidCharacterError`.